docs/docs/guides/persistence.md

```diff
@@ -54,7 +54,7 @@ Here is a basic example:
 ```jsx
 const initializeState = ({set}) => {
   for(const [key, value] of Storage.entries()) {
-    set({key}, JSON.parse(value)).value;
+    set(getAtomWithKey(key), JSON.parse(value)).value;
```

Comment:
@drarmstr where does `getAtomWithKey` come from?

Reply:
Some user-defined function.  We hesitated to even document examples for this API as it is changing.  `set()` is difficult to use here for dynamically generated nodes such as `atomFamily()` as you may not have them defined at initialization time (See #269).  Internally we use `setUnvalidatedAtomValues()`, which takes a key and does not require the atom to be defined yet.  But, that requires additional metadata in the atom to work.  You can play with it now if you like, or stay tuned for the updated API which cleans all this up.